### DISCUSSION TITLE
Connection Error am Display der Rotex HPSU

### DISCUSSION THREAD

**Discussion by Tom88s** [Trunks1982/Daikin-Rotex-HPSU-CAN#43] [General]
Guten Morgen,

leider bekomme ich immer wieder einen Connection Error am HPSU Display. (Display bleibt blau und zeigt zusätzlichen einen Schraubenschlüssel) Hat noch jemand dieses Problem?
Handelt es sich dabei um ein ähnliches Problem wie der E81? Das der Bus etwas überfordert ist und zusammenbrich?
Leider kann der Fehler immer nur durch Aus- und Einschalten gelöscht werden.

Beste Grüße

Thomas

**Comment by Trunks1982** [comment]
Nein nicht bis jetzt gehabt.Der Canbus verträgt locker 10 Abfragen gleichzeitig.

**Reply by MichaelGer1410** [reply]
Das problem kenne ich auch nicht. 
Ich habe das Model HPSU Compakt 508 "2013".

**Reply by Tom88s** [reply]
Ich habe en HPSU Compact 516 "2015"

**Reply by Trunks1982** [reply]
Ich 508 2017

**Comment by Tom88s** [comment]
Ein schlechtes Wifi Signal ist vermutlich auch nicht die Ursache…  das ist das einzige was bei mir nicht optimal ist

**Reply by Trunks1982** [reply]
ne glaube ich auch nicht.Aber bitte installier doch mal die neue bin.Und teste mal

**Reply by Tom88s** [reply]
habe jetzt die neue Rotex.bin aufgespielt und werde es mal beobachten
Ist es normal das ich nach dem update das Wlan neu einrichten muss?

**Reply by Trunks1982** [reply]
Wenn du secret in home assistant nicht nutzt ja.

**Reply by Tom88s** [reply]
Connection Error hatte ich bisher nicht mehr… aber leider wieder E81

**Reply by Trunks1982** [reply]
> Connection Error hatte ich bisher nicht mehr… aber leider wieder E81

resette mal deine Rotex.Wie gesagt bei mir wird auch jeden tag zb durch PV das WW erwärmt und alle 10s wird abgefragt und habe das Problem nicht.

**Reply by Tom88s** [reply]
Du meinst den RoConB1/U1 Reset in der Fachmann Ebene oder ?
Die Parameter müssten dabei ja alle erhalten bleiben, da Parameter Reset ein Extra Menüpunkt ist.

**Reply by Trunks1982** [reply]
Ja genau.Vielleicht hilft es ja.

**Reply by Tom88s** [reply]
Ok hab ich eben mal durchgeführt. Mal schauen ob es hilft.
Bei mir läuft der Smart Grid ebenfalls mit und macht Warmwasser mit dem PV Überschuss.

**Reply by Trunks1982** [reply]
Ich mach das mit WW Soll erhöhung.Smart Grid benutze ich zur Zeit nicht

**Reply by Tom88s** [reply]
Leider tritt der Fehler weiterhin auf :(
Jetzt bin ich etwas ratlos
Werde den ESP erst mal aus lassen
Über Fhem mit dem ELM gabs nie Probleme

**Reply by svde** [reply]
Did you try my version which only listens to messages on the canbus (it doesn't poll)?

**Reply by Tom88s** [reply]
Reicht es wenn ich die Rotex.bin von Dir Update oder muss ich den ESP komplett neu aufsetzen?

**Reply by svde** [reply]
Yaml file can be found [here](https://github.com/svde/Daikin-Rotex-HPSU-CAN/blob/main/Rotex-Daikin-CAN.SvdE.yaml).

I don't build binaries. You'll need to compile it yourself with esphome.

**Reply by Tom88s** [reply]
Leider hab ich keine Ahnung wie das geht…
Bin nicht so fit in dem Thema

**Reply by Trunks1982** [reply]
Was benutzt du für Kabel?

**Reply by Tom88s** [reply]
CAT7 Kabel twisted pair

**Reply by Trunks1982** [reply]
Mhm ok das hört sich gut an.

**Reply by Tom88s** [reply]
Wie lange sind eure Zuleitungen zum ESP?
Bei sind es ein paar Meter bis in die Unterverteilung?
Hab ich auch den Verdacht das es eventuell mit den Sperrzeiten zu tun hat… muss es aber noch genauer beobachten

**Reply by Trunks1982** [reply]
2,5 bis 3m zur Heizung.Mach Sperrzeit aus wenn möglich.Sperrzeiten hab ich nicht.

**Reply by Tom88s** [reply]
Das kann ich leider nicht machen da ich einen Wärmestromtarif habe. Da ich den Smart Grid auch benutzen möchte unterscheidet eine Siemens Logo ob das Signal vom Smart Grid kommt oder eine EVU Sperre ist. Ist es es eine EVU Sperre wird diese als Sperrzeit realisiert.
Da diese Rotex Serie normalerweise entweder den Smart Grid oder die EVU Sperre nutzen kann.

Hoffe du kannst mir halbwegs folgen

**Reply by Trunks1982** [reply]
Nur so nen bisschen.🤣

**Reply by Tom88s** [reply]
😂🙈 Die Sperrzeiten sind vorgeschrieben da ich einen vergünstigten Stromtarif für die Wärmepumpe nutze.

**Reply by Trunks1982** [reply]
Ja das kenne ich aber in der eotex kannst das ja eigentlich ausschalten und trotzdem sg nutzen

**Reply by Tom88s** [reply]
In der Rotex Steurung kann man normal nur SG oder die HT/NT Funktion nutzen.

Somit benutze ich aktuell für Rotex nur den SG.

Den Rest mach die Siemens Logo und priorisiert das Signal vom Netzbetreiber und schaltet dann zu den Sperrzeiten des Netzbetreibers in den Sperrbetrieb.

**Reply by Trunks1982** [reply]
Also geht der versorger nicht auf nt ht?
Bzw auf evu?

**Reply by Tom88s** [reply]
Ne das EVU Signal geht auf die Logo Steuerung. 
Die Rotex bekommt das Signal dann als Sperrzeit übertragen. Da das EVU Signal allen anderen SG Zuständen übergeordnet ist.

es gibt ja 4 SG Zustände

Normalbetrieb
Sperrbetrieb
PV Überschuss
Abriegelung

**Reply by Trunks1982** [reply]
Kabel abnehmen.Merkt doch keiner.

**Comment by Tom88s** [comment]
Jetzt ist es fast 3 Tage ohne Fehler durchgelaufen. Somit kann ich die den Smart Grid und Sperrzeiten auch ausschließen. 
Jetzt wird’s doof 🙈

**Reply by Trunks1982** [reply]
Ah ok.

**Comment by Tom88s** [comment]
Die lezte Idee, um den E81 weg zu bekommen ist die Leitungslänge deutlich zu reduzieren.

Bisher sind es fast 13m was mit Fhem aber nie ein Problem war…

**Reply by Trunks1982** [reply]
Sollte bei canbus wohl auch kein Problem sein.
Aber bei dem Problem bin ich echt überfragt.

**Reply by Tom88s** [reply]
![IMG_5338](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/122024828/c91834e9-937b-4852-89a7-4c39e3703c15)

Hin und wieder sind mir diese gelben Meldungen aufgefallen… ist das normal?

**Reply by Trunks1982** [reply]
Jap kannst ignorieren.

**Reply by Tom88s** [reply]
Wie funktioniert das eigentlich mit der defrost.yaml? Da wird ja zwischen Betriebsbereit und Heizen hin und her geschalten oder?
Meine Anlage ist derzeit im Sommerbetrieb nicht das da iwas nicht passt.
Sorry falls das jetzt ne dumme Frage ist…
Mit ner anderen Spannungsversorgung lief es jetzt mal 4 Tage fehlerfrei

**Reply by Trunks1982** [reply]
Defrost yaml schalte ich im Sommer ab.Die soll ja nur beim defrosten greifen.

**Comment by MichaelGer1410** [comment]
Guten Morgen, 

letzte woche Hatte ich auch einmal das Problem " Kommunikationsfehler" da half dann ein stromreset der Rotex.
Jetzt habe ich ein großes Problem, die Heizung zeigt ständig irgendwelche fehler an, 9004/7h, E78, E78, E100, E198, habe alles durch. Funktion = 0 der Heizung. Rotex Techniker kommt um 8 Uhr.
12 Jahre lief die Heizung Strörungsfrei 
In der fehlerbeschreibung bei E81 steht im Handbuch EEPROM gestört. Kann dies mit der Abfrage/Schreiben des ESP zu tun haben?

Wie funktioniert dein Programm? Werden ständig werte geschrieben oder nur abgefragt?

**Reply by Trunks1982** [reply]
Erstmal werden die nur abgefragt.
Also ich hab null Probleme.
Mach doch mal nen kompletten resett der heizung.Also eprom usw

**Reply by svde** [reply]
I had the same issue, see: [link](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/discussions/36#discussioncomment-9251161) and [link](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/discussions/36#discussioncomment-9315908).

**Comment by Tom88s** [comment]
Ich konnte das Problem mittlerweile beheben. Als erstes habe ich dem ESP32 eine separate Spannungsversorgung spendiert, da ich die Vermutung hatte das die bisherige zu schwach war.
Zudem ist mir aufgefallen das ich auf dem Schirm der Steuerleitung 13V hatte weil die Leitung EMV-technisch sehr ungünstig verlegt war. >>> Leitung anderst verlegt…

Seit dem ist es nicht mehr zu einem Fehler/Fehler E81 gekommen.

**Reply by Trunks1982** [reply]
Das freut mich das es jetzt so erstmal geholfen hat
